makers/engineer_ii.md

```diff
@@ -0,0 +1,73 @@
+Engineer II
+===========
+
+<!-- TODO: Add summary -->
+
+## Qualifications
+
+- Solid track record of writing clean, functional code with periodic guidance from other engineers.
```

Comment:
You've done I think an excellent job here articulating a set of qualifications that builds on top of Engineer I in a progressive fashion 💯

Reply:
Agree - I think these all make sense and provide great examples. I hope others on the team will find inspiration from examples like:

> Demoed a new feature at Product Review and explained how it helps meet business needs

I hadn't thought how I could better contextualize these demos and will be now